crates/edr_eth/src/transaction/request.rs

```diff
@@ -113,6 +113,7 @@ impl TransactionRequest {
 }
 
 /// A transaction request and the sender's address.
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Clone, Debug)]
```

Reply:
The other struct in this file follows the pattern `Debug, Clone`.

Reply:
I only left one comment. I thought that was your work too 😅